### PR TITLE
Add configuration option to digitalobject:load task

### PR DESCRIPTION
**Pull request by edgarrodsil**
Enable or disable update search index

### CODE REVIEW

File: lib/task/digitalObjectLoadTask.class.php [hunk 1]
```diff
@@ -72,9 +73,16 @@ public function execute($arguments = array(), $options = array())
     {
       throw new sfException('You must specify a valid filename');
     }
-
-    QubitSearch::disable();
-
+    
+    if (($options['index']))
+    {
+        QubitSearch::enable();
```
Comment (sevein) [inline lib/task/digitalObjectLoadTask.class.php]: Edgar, could you fix the indentation? We use two whitespaces as tabs.

**Comment by edgarrodsil** [comment]
I think now is well indented

File: lib/task/digitalObjectLoadTask.class.php [hunk 2]
```diff
@@ -72,9 +73,16 @@ public function execute($arguments = array(), $options = array())
     {
       throw new sfException('You must specify a valid filename');
     }
-
-    QubitSearch::disable();
-
+    
+    if (($options['index']))
```
Comment (sevein) [inline lib/task/digitalObjectLoadTask.class.php]: Sorry for nitpicking, there's four parenthesis there :)

**Comment by sevein** [comment]
Thanks @edgarrodsil! See 425fb86151b8ba002cabadd22947977d4a58f133.

**Pull request closed without merging**